test/test_parentwork.py

```diff
@@ -20,10 +20,50 @@
 import os
 import unittest
 from test.helper import TestHelper
+from mock import Mock
+import musicbrainzngs
 
 from beets.library import Item
 from beetsplug import parentwork
 
+work = {'work': {'id': '1',
+                 'title': 'work',
+                 'work-relation-list': [{'type': 'parts',
+                                         'direction': 'backward',
+                                         'work': {'id': '2'}}],
+                 'artist-relation-list': [{'type': 'composer',
+                                           'artist': {'name':
+                                                      'random composer',
+                                                      'sort-name':
+                                                      'composer, random'}}]}}
+dp_work = {'work': {'id': '2',
+                    'title': 'directparentwork',
+                    'work-relation-list': [{'type': 'parts',
+                                            'direction': 'backward',
+                                            'work': {'id': '3'}}],
+                    'artist-relation-list': [{'type': 'composer',
+                                              'artist': {'name':
+                                                         'random composer',
+                                                         'sort-name':
+                                                         'composer, random'
+                                                         }}]}}
+p_work = {'work': {'id': '3',
+                   'title': 'parentwork',
+                   'artist-relation-list': [{'type': 'composer',
+                                             'artist': {'name':
+                                                        'random composer',
+                                                        'sort-name':
+                                                        'composer, random'}}]}}
+
+
+def mock_workid_response(mbid, includes):
+    if mbid == '1':
+        return work
+    elif mbid == '2':
+        return dp_work
+    elif mbid == '3':
+        return p_work
+
 
 class ParentWorkTest(unittest.TestCase, TestHelper):
```

Comment:
Maybe call this one `ParentWorkIntegrationTest`?

Reply:
Done